Copy to SD Card root directory to update/Language Packs/language_hu.ini

```diff
@@ -1,300 +1,300 @@
+label_setting_save:Beállítások mentése
+label_setting_reset:Beállítás visszaállítása
+label_setting_restore:Beállítások újratöltése
+label_eeprom_settings:EEPROM
+label_eeprom_save_info:Mented a megváltozott\nbeállításokat az EEPROM-ba?
+label_eeprom_restore_info:Betöltöd a beállításokat az\nEEPROM-ból?
+label_eeprom_reset_info:Visszaállítod az EEPROM-ot a\ngyári beállításokra?
+label_retract_feedrate:Visszahúzás előtol.
+label_retract_length:Visszahúzás hossz
+label_retract_swap_length:Csere hossz vissza.
+label_retract_z_lift:Z emelése visszahú.
+label_retract_auto:Automata Visszahúzás
+label_recover_feedrate:Előtolás javítás
+label_swap_recover_feedrate:Csere előtolás jav.
+label_recover_length:Hossz javítás
+label_swap_recover_length:Cserehossz javítás
+label_start_print:Elindítod a %s\nfájl nyomtatását?
+label_ack_notification:ACK egyéb típus
+label_leveling_edge_distance:Ágy szélének távolsága
+label_xy_unlock:Felold XY
+label_tuning:Hangolás
+label_pid:PID
+label_pid_title:PID Automatikus Hangolás
+label_pid_start_info:A PID önhangolása elindul.\nPercekig is eltarthat.\nFolytatod?
+label_pid_start_info_2:PID Automatikus hangolása zajlik!
+label_pid_start_info_3:NE ÉRINTSD meg a kijelzőt míg nincs kész! (zöld LED jelzi)!
+label_tune_extruder:Adagoló
+label_tune_ext_extrude_100:Kiad 100mm
+label_tune_ext_temp:Adagoló hangolás | Fűt
+label_tune_ext_templow:Túl alacsony a kívánt hőfok!\nMinimum hőmérséklet: %d℃
+label_tune_ext_desiredval:A hőfok még nem érte el a kívánt értéket
+label_tune_ext_mark120mm:A 120mm-t jelöld be a szálon\nNyomd '%s' ha kész\nMérd meg a hátralévő hosszt\nadagolás után.
+label_tune_ext_heatoff:Kikapcsolod a fűtést?
+label_tune_ext_adj_esteps:Állítható E-lépés
+label_tune_ext_esteps_saved:Új E-lépés mentve!\nNe felejtsd el\nmenteni EEPROM-ba.\nÚj érték: %0.2f
+label_tune_ext_measured:Fennmaradó Hossz:
+label_tune_ext_old_estep:Régi e-lépés: %0.2f
+label_tune_ext_new_estep:Új e-lépés: %0.2f
+label_connection_settings:Kapcsolat
+label_offset_tool:Második fej eltolása
+label_hybrid_threshold:TMC Hibrid Küszöbérték
+label_notifications:Értesítések
+label_mesh_editor:Háló Szerkesztő
+label_mesh_tuner:Háló Hangoló
+label_case_light:Gépház Világítás
+label_load_started:A filament betöltése folyamatban van,\nnem lehetséges további művelet,\nmíg véget nem ér.\nKérlek várj!
+label_unload_started:A filament kiadása folyamatban van,\nnem lehetséges további művelet,\nmíg véget nem ér.\nKérlek várj!
+label_heaters_on:Néhány fűtőelem még üzemel.\nKikapcsolod a fűtőelemeket?
+label_print_finished:   Nyomtatás\n   elkészült.
+label_main_screen:Főoldal
+label_previous_print_data:Összegzés
+label_file_name:Fájl: 
+label_print_time:Nyomtatási idő: 
+label_filament_length:Szál hossza: %1.2fm
+label_filament_weight:Szál súlya: %1.2fg
+label_filament_cost:Szál költség: %1.2f
+label_no_filament_stats:Nincs szál statisztika.
+label_click_for_more:Kattints a részletekért.
+label_ext_templow:Fejhőfok alacsonyabb mint a\nminimális hőfok (%d℃).

```

Comment:
"Fejhőfok alacsonyabb, mint a\nminimális hőfok (%d℃)"
A comma is neeeded before "mint" because in this case "mint" is used to make a comparison.

Reply:
The omission is justified. I hope you are looking directly for bugs. :)

Reply:
I am with you. ;)

Reply:
Okay.